projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -523,4 +523,16 @@ public function add_settings_page_notice() {
 
 		add_action( 'admin_notices', $admin_notices );
 	}
+
+	/**
+	 * Adds Appearance menu.
+	 */
+	public function add_appearance_menu() {
+		// When the interface is set to wp-admin, we need to add a link to the Marketplace and rest of the menu keeps like core.
+		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
+			add_submenu_page( 'themes.php', esc_attr__( 'Add new theme', 'jetpack' ), __( 'Add new theme', 'jetpack' ), 'read', 'https://wordpress.com/themes/' . $this->domain );
```

Comment:
```suggestion
			add_submenu_page( 'themes.php', esc_attr__( 'Add New Theme', 'jetpack' ), __( 'Add New Theme', 'jetpack' ), 'read', 'https://wordpress.com/themes/' . $this->domain );
```
We should capitalize this for consistency with other menu items.

Reply:
Good suggestion, thanks 🎉